### PR TITLE
map api, main handlebars, and newspot handlebars

### PR DESCRIPTION
**Pull request by wooglow**

### CODE REVIEW

**Review by HibiAdelei — COMMENTED** [review]
looks good

**Review by LemonPocky — CHANGES_REQUESTED** [review]
Great start. The Google Maps API is impressive.

Next, you should figure out how to take what you've learned and apply it to newspot.handlebars. The interactive map should appear in a square on top of the page (not covering the whole page). You should also be able to get the lat/long of the marker and display it on the page.

**Review by wooglow — COMMENTED** [review]
Changed

**Review by HibiAdelei — APPROVED** [review]

File: newspot.handlebars [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<h1>New Spot</h1>
+<div id="map" style="width:50%;height:50%;"></div>
+<form class='tag-form'>
```
Comment (LemonPocky) [inline newspot.handlebars:3]: Add a title input

**Pull request closed without merging**